xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -834,7 +834,8 @@
       "thumbnail": { "type": "string" },
       "lastplayed": { "type": "string" },
       "broadcastnow": { "$ref": "PVR.Details.Broadcast" },
-      "broadcastnext": { "$ref": "PVR.Details.Broadcast" }
+      "broadcastnext": { "$ref": "PVR.Details.Broadcast" },
+      "uniqueid": { "type": "integer" }
```

Comment:
Library.Id is for the very specific Libraries Id :) Depending on the need, you can add the required true to integer and value limits See https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/json-rpc/schema/types.json#L204 for example of that.